### PR TITLE
Candida's first_week blog and Webmaker Experiement

### PR DESCRIPTION
**Pull request by TheLady**

### CODE REVIEW

File: participants/portland/candida/2014-09-10-first_day.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
+layout: post
```
Comment (kronda) [inline participants/portland/candida/2014-09-10-first_day.markdown]: ## @TheLady In order for Jekyll to see and format your post properly, you need to add --- to both the top and bottom off the header text like so: 

layout: post
title: "Zero to Portland"
date: 2014-09-10

## categories: candida

**Comment by kronda** [comment]
@TheLady The other thing I see is that you need to create a folder called _posts inside your name folder and then move all your actual blog posts into. Let's work on this together tomorrow.

**Comment by lsblakk** [comment]
One more thing - there are now two index.html files - one is in your _posts directory - can you remove that and push an update?

**Comment by lsblakk** [comment]
Oops - now the root level index.html is being deleted - that's not the one I meant.
Your folder structure should look like this:
(using ----- to try and show the levels)

candida/-
---------- | candida.patch (from the building firefox)
---------- | _drafts (optional)
---------- | images/
----------------------| image.jpg
---------- | index.html
---------- | _posts/
---------------------| 2014-09-26-an-example-post.md

**Pull request merged**